normandy/conftest.py

```diff
@@ -11,3 +14,15 @@ def api_client():
     client = APIClient()
     client.force_authenticate(user=user)
     return client
+
+
+@pytest.yield_fixture()
+def django_cache(request):
```

Comment:
This works, but I wonder if we could make a decorator instead like the database one? If not, or if that'd be a pain, or if that'd be a very un-pytesty way to do this, then don't bother and keep this.